### DISCUSSION TITLE
小程序中，如何在t-button的icon属性设置本地图片

### DISCUSSION THREAD

**Discussion by mxz2023** [Tencent/tdesign-miniprogram#2799] [Q&A]
目前icon看文档支持t-design的icon图片，填写一个图片名称就可以。如果想使用一个本地的图片，不知道是否支持？有没有能帮忙解答的？

**Comment by mxz2023** [comment]
这个是不支持么？哪里可以找到解决办法？

**Comment by anlyyao** [comment]
> 目前icon看文档支持t-design的icon图片，填写一个图片名称就可以。如果想使用一个本地的图片，不知道是否支持？有没有能帮忙解答的？

用绝对路径
